lib/sinon-chai.js

```diff
@@ -140,7 +140,9 @@
     sinonMethod("calledImmediatelyAfter", "been called immediately after %1");
     sinonMethod("calledOn", "been called with %1 as this", ", but it was called with %t instead");
     sinonMethod("calledWith", "been called with arguments %*", "%D");
+    sinonMethod("calledOnceWith", "been called exactly once with arguments %*", "%C");
     sinonMethod("calledWithExactly", "been called with exact arguments %*", "%D");
+    sinonMethod("calledOnceWithExactly", "been called exactly once with exact arguments %*", "%C");
```

Comment:
So, here's the thing about these, specifically their `nonNegatedSuffix`es.  There's really no good way to address these new ones.

Using the `", but it was called %c%C"` suffix doesn't make sense when it's only been called once.  And using `"%D"` causes some odd formatting issues in the case of multiple calls.

With `"%D"`, it ends up looking like this:
```
     AssertionError: expected spy to have been called with exact arguments A, BCall 1:
C A
A B
Call 2:
C A
B
```

I'm about 99.999% sure this is a sinon formatter issue and not an issue with sinon-chai.

Reply:
This should be resolved by this ticket: https://github.com/sinonjs/sinon/issues/1717

I'm still open to suggestions on how these `nonNegatedSuffix`es should be.

Reply:
I think just using `"%D"` should work pretty well, after that Sinon bug is fixed at least. Do you agree?